sequence_processing_pipeline/tests/data/pre_prep_w_replicates.csv

```diff
@@ -1,25 +1,25 @@
-9.20.19.RK.ST.1100	9.20.19.RK.ST.1100.A8	TRUE	AGTATATGTTTC	GTGYCAGCMGCCGCGGTAA	5	A4	A8	SF	163051748	Carmen_HOWE_KF3	109379Z	12/28/22	1266015	RNBL1950	Echo550	not applicable	not applicable	ABTX_Plate_174	ABTX_11052	description	exp design description	"Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4"	GT	Illumina	UCSDMI	3/2/23		FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Sequencing by synthesis	16S rRNA	V4	UCSDMI	Rob ABTX	Illumina MiSeq
-9.21.19.RK.ST.500	9.21.19.RK.ST.500.A10	TRUE	GGCTCGTCGGAG	GTGYCAGCMGCCGCGGTAA	5	A5	A10	SF	163051748	Carmen_HOWE_KF3	109379Z	12/28/22	1266015	RNBL1950	Echo550	not applicable	not applicable	ABTX_Plate_174	ABTX_11052	description	exp design description	"Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4"	GT	Illumina	UCSDMI	3/2/23		FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Sequencing by synthesis	16S rRNA	V4	UCSDMI	Rob ABTX	Illumina MiSeq
-9.22.19.RK.ST.700	9.22.19.RK.ST.700.A12	TRUE	GACATCTGACAC	GTGYCAGCMGCCGCGGTAA	5	A6	A12	SF	163051748	Carmen_HOWE_KF3	109379Z	12/28/22	1266015	RNBL1950	Echo550	not applicable	not applicable	ABTX_Plate_174	ABTX_11052	description	exp design description	"Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4"	GT	Illumina	UCSDMI	3/2/23		FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Sequencing by synthesis	16S rRNA	V4	UCSDMI	Rob ABTX	Illumina MiSeq
-9.16.19.RK.ST.700	9.16.19.RK.ST.700.A14	TRUE	AATTTCCTAACA	GTGYCAGCMGCCGCGGTAA	5	A7	A14	SF	163051748	Carmen_HOWE_KF3	109379Z	12/28/22	1266015	RNBL1950	Echo550	not applicable	not applicable	ABTX_Plate_174	ABTX_11052	description	exp design description	"Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4"	GT	Illumina	UCSDMI	3/2/23		FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Sequencing by synthesis	16S rRNA	V4	UCSDMI	Rob ABTX	Illumina MiSeq
-9.16.19.RK.ST.1100	9.16.19.RK.ST.1100.A16	TRUE	ATAAACGGACAT	GTGYCAGCMGCCGCGGTAA	5	A8	A16	SF	163051748	Carmen_HOWE_KF3	109379Z	12/28/22	1266015	RNBL1950	Echo550	not applicable	not applicable	ABTX_Plate_174	ABTX_11052	description	exp design description	"Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4"	GT	Illumina	UCSDMI	3/2/23		FWD:GTGYCAGCMGCCGCGGTAA; REV:GGACTACNVGGGTWTCTAAT	Sequencing by synthesis	16S rRNA	V4	UCSDMI	Rob ABTX	Illumina MiSeq
+	orig_name	sample_name	contains_replicates	barcode	primer	primer_plate	well_id_96	well_id_384	plating	extractionkit_lot	extraction_robot	tm1000_8_tool	primer_date	mastermix_lot	water_lot	processing_robot	tm300_8_tool	tm50_8_tool	sample_plate	project_name	well_description	experiment_design_description	library_construction_protocol	linker	platform	run_center	run_date	run_prefix	pcr_primers	sequencing_meth	target_gene	target_subfragment	center_name	center_project_name	instrument_model	runid	tm10_8_tool
```

Comment:
Also, just to confirm, is it OK that tm10_8_tool is an empty column?

Reply:
It is okay, but while I had it open I copied the 'not applicable' string from one of the other tm_ columns for consistency.